k8s/view-api/prod-view-api.yaml

```diff
@@ -70,38 +70,8 @@ spec:
           lifecycle:
             preStop:
               exec:
-                command: [ "sh", "-c", "sleep 10" ]
-        - name: views-api
```

Comment:
[rollout support for 'views-api'](https://github.com/airqo-platform/AirQo-api/pull/1062/commits/df8cd5c1db242f7308433d940c0c8d77f3dd3f9f)